hc/front/tests/test_update_timeout.py

```diff
@@ -57,3 +57,29 @@ def test_it_checks_ownership(self):
         self.client.login(username="charlie@example.org", password="password")
         r = self.client.post(url, data=payload)
         assert r.status_code == 403
+
+    def test_it_works(self):
```

Comment:
@pndemo isn't this test similar to the first test in this file - https://github.com/andela/hc-kalii/blob/0d2bec48f8577eda670a3ca99346fa21f982e5a0/hc/front/tests/test_update_timeout.py#L12